config/benchmarks.json

```diff
@@ -11,6 +11,11 @@
             "templates": ["treats"]
         }
     ],
+    "DrugCentral_subset": [
+        {
+            "source": "DrugCentral_subset",
+            "templates": ["treats"]
+        }
```

Comment:
That error is happening because we're missing a closing square bracket and comma here.